From: Gregor Jasny <gjasny@googlemail.com>
Description: Prevent autoconf from mangling passed cflags
 These flags are passed by dpkg-buildflags for hardening.
Forwarded: not-needed

--- a/configure.ac
+++ b/configure.ac
@@ -147,8 +147,8 @@
 
 CARES_CHECK_COMPILER
 CARES_SET_COMPILER_BASIC_OPTS
-CARES_SET_COMPILER_DEBUG_OPTS
-CARES_SET_COMPILER_OPTIMIZE_OPTS
+#CARES_SET_COMPILER_DEBUG_OPTS
+#CARES_SET_COMPILER_OPTIMIZE_OPTS
 CARES_SET_COMPILER_WARNING_OPTS
 
 if test "$compiler_id" = "INTEL_UNIX_C"; then
